core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -621,12 +626,18 @@ public void reload() {
      *
      * @return true if the version number is not 'DEV'.
      */
+    @SuppressWarnings("BooleanMethodIsAlwaysInverted")
     public boolean isProductionEnvironment() {
         // First is if Blossom runs, second is if Blossom doesn't run
         //noinspection ConstantConditions,MismatchedStringCase - changes in production
         return !("git-local/dev-0000000".equals(GeyserImpl.GIT_VERSION) || "${gitVersion}".equals(GeyserImpl.GIT_VERSION));
     }
 
+    public boolean isDevBuild() {
+        //noinspection ConstantValue - changes in production
+        return !GeyserImpl.BRANCH.equals("master") && !GeyserImpl.REPOSITORY.equalsIgnoreCase("https://github.com/GeyserMC/Geyser");
```

Comment:
Why are the variables fully qualified here? They can just be BRANCH and REPOSITORY?